python/GangaDirac/Lib/Utilities/DiracUtilities.py

```diff
@@ -214,6 +214,34 @@ def _checkProxy( delay=60, renew = True, shouldRaise = True, force = False ):
             last_modified_time = time.time()
 
 
+def getAccessURLs(lfns):
```

Comment:
I'm not sure that `DiracUtilities.py` is the right place for this. This file is used for functions which support the DIRAC subsystem, not for miscellaneous DIRAC functions.

Given that this function is only used in one place (in a one-liner no less) inside `DiracBase`, it probably makes more sense to put it in inline inside `getOutputDataAccessURLs`.